stdlib/src/gc.rs

```diff
@@ -6,22 +6,52 @@ mod gc {
 
     #[pyfunction]
     fn collect(_args: FuncArgs, _vm: &VirtualMachine) -> i32 {
-        0
+        #[cfg(feature = "gc")]
+        {
+            usize::from(rustpython_vm::object::collect()) as i32
+        }
+        #[cfg(not(feature = "gc"))]
+        {
+            0
+        }
     }
 
     #[pyfunction]
     fn isenabled(_args: FuncArgs, _vm: &VirtualMachine) -> bool {
-        false
+        #[cfg(feature = "gc")]
+        {
+            rustpython_vm::object::isenabled()
+        }
+        #[cfg(not(feature = "gc"))]
+        {
+            false
+        }
     }
 
     #[pyfunction]
     fn enable(_args: FuncArgs, vm: &VirtualMachine) -> PyResult {
-        Err(vm.new_not_implemented_error("".to_owned()))
+        #[cfg(feature = "gc")]
+        {
+            rustpython_vm::object::enable();
+            Ok(vm.new_pyobj(true))
+        }
+        #[cfg(not(feature = "gc"))]
+        {
+            Err(vm.new_not_implemented_error("".to_owned()))
+        }
     }
 
     #[pyfunction]
     fn disable(_args: FuncArgs, vm: &VirtualMachine) -> PyResult {
-        Err(vm.new_not_implemented_error("".to_owned()))
+        #[cfg(feature = "gc")]
+        {
+            rustpython_vm::object::disable();
```

Comment:
> Also the `gc` module could probably get moved to rustpython-vm proper.

Indeed, in CPython, the `gc` module is compiled into the interpreter (rather than being dynamically linked):

```text
Python 3.11.0 (main, Oct 25 2022, 13:57:33) [Clang 14.0.0 (clang-1400.0.29.202)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import gc
>>> gc.__file__
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'gc' has no attribute '__file__'. Did you mean: '__name__'?
```

Reply:
> It would be good for these to have more descriptive names, either a `gc_` prefix or a separate module. Also the _gc module could probably get moved to rustpython-vm proper.

how about "gc_bacon" as feature name(as bacon is the papers' author)?